src/main/java/org/fundacionjala/sfdc/pageobjects/SalesForceMainTabClassic.java

```diff
@@ -0,0 +1,98 @@
+package org.fundacionjala.sfdc.pageobjects;
+
+
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+/**
+ * SalesForceMainTabClassic.java
+ * Class of Salesforce main tab that contains all the options.
+ */
+public class SalesForceMainTabClassic extends SalesForceConnection {
+    private By tabButton;
+    private By accountsLink;
+    private WebDriverWait wait;
+
+    /**
+     * Constructor.
+     * @param url web page.
+     * @param actionButton menu button.
+     */
+    public SalesForceMainTabClassic(final String url, final String actionButton) {
+        super();
+        this.tabButton = By.cssSelector(actionButton);
+        this.accountsLink = By.cssSelector(url);
+    }
+
+    /**
+     * Getter of tabButton.
+     * @return the button.
+     */
+    public WebElement getTabButton() {
+        return this.webDriver.findElement(this.tabButton);
+    }
+
+    /**
+     * Getter of accountsLink.
+     * @return the link.
+     */
+    public WebElement getAccountsLink() {
+        return this.webDriver.findElement(this.accountsLink);
+    }
+
+    /**
+     * Method to click tabButton.
+     * @param button to click.
+     */
+    public void clickTabButton(final WebElement button) {
+        button.click();
+    }
+
+    /**
+     * Method to click accountsLink.
+     * @param link to click.
+     */
+    public void clickAccountsLink(final WebElement link) {
+        link.click();
+    }
+
+    /**
+     * Method to display all the options in Salesforce.
+     */
+    public void displayOptions() {
+        WebElement button = getTabButton();
+        clickTabButton(button);
+        wait = DriverManager.getInstance().getWaiter();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)